en/django_forms/README.md

```diff
@@ -420,7 +420,7 @@ $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets.)
```

Comment:
Also questioning the parenthesis here

Reply:
Agreed that the parens should be removed.